src/time-slot-bar/time-slot-bar.tsx

```diff
@@ -148,26 +186,25 @@ export const TimeSlotBar = ({
         const slotWidth = TimeSlotBarHelper.calculateWidth(
             startTime,
             endTime,
-            CELL_WIDTH
+            cellWidth
         );
+
+        const isClickable = !!onClick && variant === "default";
```

Comment:
So it is clickable only if you passed in onClick and it is default variant? Or actually if it is default variant it is already clickable?